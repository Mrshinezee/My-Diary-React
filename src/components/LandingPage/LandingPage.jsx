// react libraries
import React, { Fragment } from 'react';

/**
 * @desc renders Landing
* @returns {object} Landing
 */
const Landing = () => (
  <Fragment>
     <div className="menu-links">
          <div id="menu-links-left">
            <span><a href="index.html">MY-DIARY</a></span>
          </div>
    
          <div className="menu-icon">
            <label htmlFor="menu-check">
              <span></span>
              <span></span>
              <span></span>
            </label>
          </div>
    
          <input type="checkbox" id="menu-check" />
          <div className="menu-links-right">
            <a href="sign-up.html">Sign Up</a>
            <a href="login.html">Login</a>
          </div>      
    </div>
    
      <header id="header">
        <div className="container">
          <div id="topic">
          </div>
          <h5>Pen down your thoughts and feelings</h5>
          <button className="btn-signup" >Get Started</button>
        </div>
      </header>

      <section className="grid">
        <div className="container">
          <div className="box">
            <img src="https://d3c2plo0qyv3hc.cloudfront.net/images/landing/feature-icon__reminder.png" alt="" />
            <h3>Never Forget To Write</h3>
            <p>Custom email reminders help you make sure you never forget to write</p>
          </div>
          <div className="box">
            <img src="https://d3c2plo0qyv3hc.cloudfront.net/images/landing/feature-icon__mobile.png" alt="" />
            <h3>Write From Anywhere</h3>
            <p>You can access your diary from anywhere in the world</p>
          </div>
          <div className="box">
            <img src="https://d3c2plo0qyv3hc.cloudfront.net/images/landing/feature-icon__security2.png" alt="" />
            <h3>Security</h3>
            <p>You can be rest assured that your entries are totaly safe and secured </p>
          </div>
        </div>
      </section>

      <footer>
        <p>My-Diary, Copyright &copy; 2018</p>
      </footer>
  </Fragment>
);

export default Landing;
