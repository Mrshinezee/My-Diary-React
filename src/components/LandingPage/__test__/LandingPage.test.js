// React Libraries
import React from 'react';

// third-party libraries
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


// modules importation
import LandingPage from '../LandingPage';


configure({
  adapter: new Adapter(),
});

// SnapShot Test
describe('Renders the Landing page', () => {

  it('should render the landing page', () => {
    const component = shallow(
      <LandingPage />
    );
    expect(component).toMatchSnapshot();
  });
});