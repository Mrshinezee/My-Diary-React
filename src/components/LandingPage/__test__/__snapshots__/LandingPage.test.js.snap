// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Landing page should render the landing page 1`] = `
<Fragment>
  <div
    className="menu-links"
  >
    <div
      id="menu-links-left"
    >
      <span>
        <a
          href="/"
        >
          MY-DIARY
        </a>
      </span>
    </div>
    <div
      className="menu-icon"
    >
      <label
        htmlFor="menu-check"
      >
        <span />
        <span />
        <span />
      </label>
    </div>
    <input
      id="menu-check"
      type="checkbox"
    />
    <div
      className="menu-links-right"
    >
      <a
        href="register"
      >
        Sign Up
      </a>
      <a
        href="login"
      >
        Login
      </a>
    </div>
  </div>
  <header
    id="header"
  >
    <div
      className="container"
    >
      <div
        id="topic"
      />
      <h5>
        Pen down your thoughts and feelings
      </h5>
      <button
        className="btn-signup"
      >
        Get Started
      </button>
    </div>
  </header>
  <section
    className="grid"
  >
    <div
      className="container"
    >
      <div
        className="box"
      >
        <img
          alt=""
          src="https://d3c2plo0qyv3hc.cloudfront.net/images/landing/feature-icon__reminder.png"
        />
        <h3>
          Never Forget To Write
        </h3>
        <p>
          Custom email reminders help you make sure you never forget to write
        </p>
      </div>
      <div
        className="box"
      >
        <img
          alt=""
          src="https://d3c2plo0qyv3hc.cloudfront.net/images/landing/feature-icon__mobile.png"
        />
        <h3>
          Write From Anywhere
        </h3>
        <p>
          You can access your diary from anywhere in the world
        </p>
      </div>
      <div
        className="box"
      >
        <img
          alt=""
          src="https://d3c2plo0qyv3hc.cloudfront.net/images/landing/feature-icon__security2.png"
        />
        <h3>
          Security
        </h3>
        <p>
          You can be rest assured that your entries are totaly safe and secured 
        </p>
      </div>
    </div>
  </section>
  <footer>
    <p>
      My-Diary, Copyright Â© 2018
    </p>
  </footer>
</Fragment>
`;
